internal/bootfiles/bootfiles.go

```diff
@@ -55,3 +60,30 @@ func (b *BootFiles) DownloadBootFiles(ctx context.Context, params operations.Dow
 	return filemiddleware.NewResponder(operations.NewDownloadBootFilesOK().WithPayload(reader),
 		objectName, contentLength)
 }
+
+func (b *BootFiles) UploadBootFiles(ctx context.Context, params operations.UploadBootFilesParams) middleware.Responder {
+	log := logutil.FromContext(ctx, b.log)
+
+	// Create/Update version in DB
+	version := *params.UploadBootFilesParams.OpenshiftVersion
+	tx := b.db.Begin()
+	if err := tx.Save(&version).Error; err != nil {
+		err = errors.Wrapf(err, "Error storing OCP and RHCOS versions in DB")
+		log.Error(err)
+		return common.GenerateErrorResponder(err)
+	}
+	if err := tx.Commit().Error; err != nil {
+		return common.GenerateErrorResponder(err)
+	}
+
+	// Upload relevant boot files according to the specified version
+	ocpVersion := *params.UploadBootFilesParams.OpenshiftVersion.DisplayName
```

Comment:
Wouldn't we want to update the DB data only after the upload succeeded? Otherwise, we might end with a malformed DB.
In addition, I think we should add a `defer` function we do in the inventory for transactions
https://github.com/openshift/assisted-service/blob/e400a0e43bc9f979e48809c8b5cc02262ee4c9eb/internal/bminventory/inventory.go#L2461-L2470

Reply:
changed impl, no longer storing in db.